helm/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 name: sagemaker-chart
 description: A Helm chart for the ACK service controller for Amazon SageMaker (SageMaker)
-version: v1.1.1
-appVersion: v1.1.1
```

Comment:
I don't understand the need of this change? moving backwards?

Reply:
Avoid cutting a patch release. We will be cutting a minor release with another change.

Reply:
Gotcha! thank you :)